sql/information_schema/information_schema.go

```diff
@@ -810,7 +809,7 @@ func tablesRowIter(ctx *Context, cat Catalog) (RowIter, error) {
 				nil,                        // max_data_length
 				nil,                        // max_data_length
 				nil,                        // data_free
-				autoVal,                    // auto_increment
+				nil,                        // auto_increment (always nil)
```

Comment:
This isn't right, MySQL definitely returns accurate info for this. Why do we need to get rid of it?

Reply:
mysql always returns NULL for user space tables